learning-resources.md

```diff
@@ -0,0 +1,21 @@
+# Programming resources
+
+- [Get Coding!](http://getcodingkids.com)
+- [Code Academy](https://www.codecademy.com/)
+- [Free Code Camp](https://www.freecodecamp.org/)
+- [RoboCode](http://robowiki.net/wiki/Robocode)
+- [WIT Java Tutorials](https://tycourses.github.io/intro-to-coding/index.html)
+
+
+# Admin resources
+
+- [Tutors NPM (WIT static site generation module)](https://www.npmjs.com/package/tutors)
+
+
+# Communities
```

Comment:
What do you think of adding the waterford tech meetup to this list?

Reply:
Think this is a good idea, but perhaps may require some form of consent/ notification to parents, given it's an adult environment w/ beer ?